### PR TITLE
move tar-changes into a rdopkg_tar python module and add a setup.py

### PR DESCRIPTION
**Pull request by andrewschoen**
This allows for rdopkg-tar to be installed inside of a virtualenv. Running the `rdokpg-tar` command inside the virutalenv is the same as running `tar-changes` directly.

### CODE REVIEW

**Comment by ktdreyer** [comment]
This is great, thanks. I think it will make unit testing easier too.

Can we get rid of the `tar-changes` script, do you think? Then the code only lives in one place.

**Comment by andrewschoen** [comment]
> This is great, thanks. I think it will make unit testing easier too.
> 
> Can we get rid of the `tar-changes` script, do you think? Then the code only lives in one place.

Fixed. I'd actually done that locally and just forgot to commit it.

**Comment by ktdreyer** [comment]
It occurred to me that we're going to break the Jenkins jobs that expect to run this script

**Comment by ktdreyer** [comment]
Looks like we can resolve this by making `tar-changes` a symlink, like:

```
ln -s rdopkg_tar/tar_changes.py tar-changes
```

Want to add that to this PR? Then we can: merge this, then migrate the Jenkins jobs to stop using the symlink, then drop the symlink.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        console_scripts = [
+            'rdopkg-tar = rdopkg_tar.tar_changes:main',
+        ],
+    ),
+    classifiers = [
+        'Environment :: Console',
+        'Intended Audience :: Information Technology',
+        'Intended Audience :: System Administrators',
+        'Operating System :: POSIX :: Linux',
+        'License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)',
```
Comment (ktdreyer) [inline setup.py:30]: Let's use ASL 2.0 here. I've added it in https://github.com/red-hat-storage/rdopkg-tar/pull/17

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.5',
```
Comment (ktdreyer) [inline setup.py:33]: Let's take out Python 2.7 and 3.5 here since we don't test it.

**Comment by ktdreyer** [comment]
After discussing this with Andrew, here is our success criteria for this code change:

We need to be able to continue to execute this script outside any virtualenv, like:

```
./tar-changes --help
```

With this PR, we also expect to be able to run the `rdopkg-tar` command within a virtualenv, like:

```
virtualenv venv
. venv/bin/activate
python setup.py develop
rdopkg-tar --help
```

**Pull request closed without merging**